assembl/static2/js/app/components/harvesting/harvestingBox.jsx

```diff
@@ -225,9 +233,60 @@ class DumbHarvestingBox extends React.Component<Object, Props, State> {
       });
   };
 
+  showValidatedHarvesting = (nature: ?string, action: ?string) => {
+    if (nature && action) {
+      return <div className="harvesting-taxonomy-label">{`${I18n.t(`search.taxonomy_nature.${nature}`)} + ${I18n.t(`search.taxonomy_action.${action}`)}`}</div>;
+    } else if (nature) {
+      return <div className="harvesting-taxonomy-label">{I18n.t(`search.taxonomy_nature.${nature}`)}</div>;
+    }
+    return action ? <div className="harvesting-taxonomy-label">{I18n.t(`search.taxonomy_action.${action}`)}</div> : null;
```

Comment:
why "simply" ? action may be `null`/`undefined`, no ?

Reply:
Correct